backend/models/User.js

```diff
@@ -10,29 +12,34 @@ const userSchema = new Schema({
   },
   email: {
     type: String,
-    required: true,
+    required: false, // Changed to false
     unique: true,
     trim: true,
     lowercase: true,
     match: [/^\S+@\S+\.\S+$/, "Please enter a valid email address"],
   },
   password: {
     type: String,
-    required: true,
+    required: false, // Changed to false
```

Comment:
ise false ku kr diya ye to required hi sahi rhega n ??

Reply:
Yes sir but Google authentication ke time pe error de raha tha tbhi try Kiya false krke tb chal raha tha

Reply:
but ye to fix se ek bug bn gya then iska kya ?

Reply:
Will change according to the new model as discussed in meeting now.